docs/guides/blocks/column-formats.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Column Formats
+---
+
+# Add custom column formats
+
+A column format is a custom column type that you apply to any column in any Coda table. It changes how the values within that column are interpreted and displayed, while still allowing users to quickly edit the underlying value.
```

Comment:
Total nit, `that you can apply` is a tad clearer IMO, ditto on the example page.

Reply:
Good catch, that was a typo.